index.html

```diff
@@ -3262,6 +3306,36 @@ <h5>Existing implementations</h5>
 <dt>bing-maps-embed</dt>
 <dd><a>partial support</a>: does not support markers for a single location, but it is possible to display a map centered around a particular street address with no marker</dd>
 </dl>
+
+
+<h5>Uses beyond mapping</h5>
+<p>
+There are many cases where a website may expect an address from a user.
+An input type specifically for addresses has a wide variety applications other than displaying a map,
+such as reducing user error when facilitating e-commerce and delivery services.
+</p>
+
+<h5>Conclusion</h5>
+<!-- Cost, benefits, and recommendation on whether this should be a requirement.
+Should include an `<a>` element with attribute `data-ucr-role="conclusion"`
+whose text content or `data-lt` value matches one of the conclusions (requirement, enhancement, impractical, undecided).
+See examples.
+-->
+<p>
+Based on the limited data, we are <a data-ucr-role="conclusion">undecided</a> about whether this should be a requirement.
+</p>
+<ul data-ucr-role="conclusion-notes">
+<li>
+<a data-ucr-role="tag" href="#i18n-difficult"></a>
+There are significant regional differences when it comes to how addresses are formatted.
+A good implementation of this would need to take these regional variations into account.
+</li>
+<li>
+<a data-ucr-role="tag" href="#privacy-personal-data"></a>
+It is unlikely that this capability could be supported in a fully client-side way,
+the inputted address may be revealed to the browser or website author.
```

Comment:
This probably needs re-writing to be more clear. (But I'm OK merging it as is, and I'll give it a go on a future editing pass.)